website/docs/r/integration_ghcr.html.markdown

```diff
@@ -0,0 +1,50 @@
+---
+subcategory: "Container Registry Integrations"
+layout: "lacework"
+page_title: "Lacework: lacework_integration_ghcr"
+description: |-
+Create and manage Github Container Registry (GHCR) integrations
+---
+
+# lacework\_integration\_ghcr
+
+Use this resource to integrate a Github Container Registry (GHCR) with Lacework to assess, identify,
+and report vulnerabilities found in the operating system software packages in a Docker container
+image. For more information, see the [Integrate Github Container Registry documentation](https://support.lacework.com/hc/en-us/articles/1500002482302-Integrate-GitHub-Container-Registry).
+
+## Example Usage
+
+```hcl
+resource "lacework_integration_ghrc" "example" {
+  name            		 = "My Github registry Registry"
+  username        		 = "my-user"
+  password        		 = "a-secret-password"
+  ssl             		 = true
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The integration name.
+* `username` - (Required) The Github username.
+* `password` - (Required) The Github personal access token with permission read:packages.
+* `ssl` - (Optional) Enable or disable SSL communication. Defaults to `true`.
+* `registry_notifications` - (Optional) Subscribe to registry notifications. Defaults to `false`.
+* `enabled` - (Optional) The state of the external integration. Defaults to `true`.
+* `limit_num_imgs` - (Optional) The maximum number of newest container images to assess per repository. Must be one of `5`, `10`, or `15`. Defaults to `5`.
+* `limit_by_tags` - (Optional) A list of image tags to limit the assessment of images with matching tags. If you specify `limit_by_tags` and `limit_by_label` limits, they function as an `AND`.
+* `limit_by_label` - (Optional) A list of key/value labels to limit the assessment of images. If you specify `limit_by_tags` and `limit_by_label` limits, they function as an `AND`.
```

Comment:
There is a block of information missing that explains the format of the `limit_by_label` argument. 

```
The `limit_by_label` block can be defined multiple times to define multiple label limits, it supports:
* `key` - (Required) The key of the label.
* `value` - (Required) The value of the label.
```